backend/src/server/routes/v2/project-router.ts

```diff
@@ -127,29 +136,24 @@ export const registerProjectRouter = async (server: FastifyZodProvider) => {
     },
     schema: {
       body: z.object({
-        projectName: z.string().trim(),
-        slug: z
-          .string()
-          .min(5)
-          .max(36)
-          .refine((v) => slugify(v) === v, {
-            message: "Slug must be a valid slug"
-          })
-          .optional(),
-        organizationId: z.string().trim()
+        projectName: z.string().trim().describe("Name of the project you're creating"),
+        slug: slugSchema
+          .optional()
+          .describe("An optional slug for the project. If not provided, it will be auto-generated"),
+        organizationSlug: z.string().trim().describe("The slug of the organization to create the project in")
       }),
       response: {
         200: z.object({
           project: projectWithEnv
         })
       }
     },
-    onRequest: verifyAuth([AuthMode.JWT, AuthMode.API_KEY, AuthMode.IDENTITY_ACCESS_TOKEN]),
+    onRequest: verifyAuth([AuthMode.JWT, AuthMode.IDENTITY_ACCESS_TOKEN]),
     handler: async (req) => {
       const project = await server.services.project.createProject({
         actorId: req.permission.id,
         actor: req.permission.type,
-        orgId: req.body.organizationId,
+        orgSlug: req.body.organizationSlug,
```

Comment:
I thought this would be for a new project end point and not the existing one

Reply:
Oh right, no one other than the web UI uses this end point because it was not possible to create projects and invite users programmatically